### ISSUE TITLE
[Utility][Document] Design QoS data structures

### ISSUE THREAD

**Issue by jessicadaugherty** [pokt-network/pocket#713]
## Objective

Design and implement quality of service (QoS) protobufs for Servicers in the utility module to unblock Fisherman interface design. Ensure that the protobufs capture relevant information about Servicers, focusing on the registration process and QoS metrics.

## Origin Document

[V1 Utility Specification - Servicer Protocol
](https://github.com/pokt-network/pocket-network-protocol/tree/main/utility#321-staking)

## Goals

- Design QoS protobufs to capture relevant information about Servicers, such as PublicKey, StakeAmount, ServiceURL, RelayChains, GeoZone, and optional OperatorPublicKey
- Ensure that the protobufs facilitate proper measurement of Network SLA (Service Level Agreement) metrics, including Availability, Latency, and Data Accuracy

## Deliverable

- [ ] Design and implement QoS protobufs for Staked Servicers
- [ ] Ensure that the protobufs support measurement of Network SLA metrics (Availability, Latency, and Data Accuracy) for Servicers

## Non-goals / Non-deliverables

- Designing protobufs for other actors besides Servicers
- Implementing other servicer protocol functionality

## General issue deliverables

- [ ] Update the appropriate CHANGELOG
- [ ] Update any relevant READMEs (local and/or global)
- [ ] Update any relevant global documentation & references
- [ ] If applicable, update the source code tree explanation
- [ ] If applicable, add or update a state, sequence or flowchart diagram using [mermaid](https://mermaid-js.github.io/mermaid/)

## Testing Methodology

- [ ] **All tests**: `make test_all`
- [ ] **LocalNet**: verify a `LocalNet` is still functioning correctly by following the instructions at [docs/development/README.md](https://github.com/pokt-network/pocket/tree/main/docs/development)

---

**Creator**: @jessicadaugherty

**Comment by jessicadaugherty** [comment]
Report cards out of scope?

**Comment by Olshansk** [comment]
Going to close this out in exchange for the process being put together in #733.
